api-reference/10 UI Widgets/dxHtmlEditor/3 Methods/getModule(moduleName).md

```diff
@@ -3,13 +3,15 @@ id: dxHtmlEditor.getModule(moduleName)
 ---
 ---
 ##### shortDescription
-<!-- Description goes here -->
+Gets instance of a [module](https://github.com/DevExpress/devextreme-quill/tree/master/modules). 
 
 ##### return: Object
-<!-- Description goes here -->
+A module's instance. 
 
 ##### param(moduleName): String
-<!-- Description goes here -->
+The name of a [registered](/Documentation/ApiReference/UI_Widgets/dxHtmlEditor/Methods/#registercomponents) module. 
 
 ---
-<!-- Description goes here -->
\ No newline at end of file
+You can use DevExtreme Quill modules, DevExtreme UI [modules](https://github.com/DevExpress/DevExtreme/tree/20_2/js/ui/html_editor/modules) or create custom ones.
```

Comment:
External links should open in a new tab. Also, the link to Quill modules looks better here. I would also suggest changing the text a bit:

> You can get DevExtreme Quill modules, DevExtreme UI modules, or custom modules.